services/common/src/components/common/Loading.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @constant Loading is a full page loading spinner
+ */
+import React from "react";
+import { useSelector } from "react-redux";
+import { getSystemFlag } from "@mds/common/redux/selectors/authenticationSelectors";
+import { SystemFlagEnum } from "@mds/common/constants/enums";
+import Lottie from "react-lottie";
+
+import coreLoader from "@mds/common/assets/coreLoader.json";
+import mineSpaceLoader from "@mds/common/assets/mineSpaceLoader.json";
+
+const Loading = () => {
+  const systemFlag = useSelector(getSystemFlag);
+  const isCore = systemFlag === SystemFlagEnum.core;
+  const defaultOptions = {
+    loop: true,
+    autoplay: true,
+    animationData: isCore ? coreLoader : mineSpaceLoader,
+  };
+  return (
+    <div id="loading-screen">
+      <div id="loader">
+        <Lottie options={defaultOptions} />
```

Comment:
This looks a little strange to me!
It looks like the coreLoader/mineSpaceLoader are SVGs pretending to be JSON files.
Lottie also gives us a million deprecation warnings.
What if we just use the antd `<Spin />` like we have in `LoadingWrapper.js`? Currently it gets passed the colour of the spinner, but that seems a lot simpler as the change variable between MS/CORE (assuming that we can't just CSS it)

Reply:
☝️ Also vote for not introducing lottie again, it's a big library and way overkill for a loading-spinner 😅

Reply:
Sounds good, lottie is taken out now and uses the antd `<Spin />`